README.md

```diff
@@ -129,20 +129,21 @@ nationality = user_profile.get('nationality')
 
 ## Running the Examples
 
-The callback URL for both example projects will be http://localhost:5000/yoti/auth/
+The callback URL for both example projects will be ``http://localhost:5000/yoti/auth/``
 
 ### With Docker
 
-#### Flask
-
-To run the Flask container:
+To run the Flask or Django containers:
  
 1. Clone this repository
-1. Change directory to the Flask example project with `cd examples/yoti_example_flask`
+1. Change directory to the example project directory with
+   * `cd examples/yoti_example_flask` for __Flask__   
+   __OR__
+   * `cd examples/yoti_example_django` for __Django__
 1. Make sure the environment variables `YOTI_APPLICATION_ID`, `YOTI_CLIENT_SDK_ID` and `YOTI_KEY_FILE_PATH` are set (instructions in the [Configuration](#configuration) section). _Please note that with Docker, the .pem file must reside in a location within where docker is being run from, so it should be placed somewhere under the respective  [yoti_example_flask](/examples/yoti_example_flask)/[yoti_example_django](/examples/yoti_example_django) folders._
 1. Rebuild the images if you have modified the docker-compose.yml file with
    - `docker-compose build --no-cache`
-1. Build the container with `docker-compose up`
+1. Start the container with `docker-compose up`
```

Comment:
This will work fine, but I kind of like the detached mode `docker-compose up -d` - just a preference.

Reply:
I like to see the logs in the terminal!

Reply:
I prefer it without the `-d` 😏